airgun/views/new_contentview.py

```diff
@@ -0,0 +1,262 @@
+from wait_for import wait_for
+from widgetastic.utils import ParametrizedLocator
+from widgetastic.widget import Checkbox
+from widgetastic.widget import ParametrizedView
+from widgetastic.widget import Text
+from widgetastic.widget import TextInput
+from widgetastic.widget import View
+from widgetastic_patternfly import BreadCrumb
+from widgetastic_patternfly import Tab
+from widgetastic_patternfly4.ouia import Button as PF4Button
+from widgetastic_patternfly4.ouia import PatternflyTable
+from widgetastic_patternfly4.ouia import Switch
+
+from airgun.views.common import BaseLoggedInView
+from airgun.views.common import LCESelectorGroup
+from airgun.views.common import NewAddRemoveResourcesView
+from airgun.views.common import SearchableViewMixinPF4
+from airgun.widgets import ActionsDropdown
+from airgun.widgets import ConfirmationDialog
+from airgun.widgets import EditableEntry
+from airgun.widgets import PF4Search
+from airgun.widgets import ProgressBarPF4
+from airgun.widgets import ReadOnlyEntry
+
+class NewContentViewTableView(BaseLoggedInView, SearchableViewMixinPF4):
+    title = Text("//h1[contains(., 'Content Views')]")
+    create_content_view = PF4Button('OUIA-Generated-Button-primary-1')
+    table = PatternflyTable(
+        component_id='OUIA-Generated-Table',
```

Comment:
Did Search works for you for me it did not ? I think we have to remove the `OUIA-Generated-Table'` inherit this table with our own table and add custom read()method. One way to do that is to self.read() after that remove the non required attributes.